src/pke/unittest/utckksrns/UnitTestCompositeScalingBootstrap.cpp

```diff
@@ -147,6 +147,9 @@ static std::vector<TEST_CASE_UTCKKSRNSCS_BOOT> testCases = {
     // TestType,     Descr, Scheme,          RDim, MultDepth,  SModSize,     DSize, BatchSz, SecKeyDist,      MaxRelinSkDeg, FModSize,  SecLvl,       KSTech, ScalTech,        LDigits,      PtMod, StdDev, EvalAddCt, KSCt, MultTech, EncTech, PREMode, multipartyMode, decryptionNoiseMode, executionMode, noiseEstimate, registerWordSize, LvlBudget, Dim1,     Slots
```

Comment:
please rename UnitTestCompositeScalingBootstrap.cpp. It should be UnitTesCKKSrnstCompositeScalingBootstrap.cpp as this is a test file for CKKS

Reply:
Will do.